Components/toast.css

```diff
@@ -0,0 +1,59 @@
+.btn-toast-1 {
+  cursor: pointer;
+  margin-bottom: var(--margin-width);
+}
+.toast-desc-1,
+.toast-desc-2 {
+  visibility: hidden;
+  border: 1px solid var(--white-color);
+  padding: 1rem;
+  background-color: var(--toast-background-color);
+  width: 40%;
+  text-align: center;
+  left: 30%;
+  border-radius: 5px;
+  position: fixed;
+  top: 80px;
+  z-index: 100;
```

Comment:
you can keep track of the z-indexes so that in future you can avoid the magic num. like 999 etc!

Reply:
yeah I don't know when I gave it as 100, but now I have resolved it